docs/modules/ROOT/pages/server/environment-repository/vault-backend.adoc

```diff
@@ -158,3 +158,63 @@ Properties written to `secret/application` are available to <<_vault_server,all
 An application with the name, `myApp`, would have any properties written to `secret/myApp` and `secret/application` available to it.
 When `myApp` has the `dev` profile enabled, properties written to all of the above paths would be available to it, with properties in the first path in the list taking priority over the others.
 
+[[decrypting-vault-secrets]]
+== Decrypting Vault Secrets in Property Sources
+
+Spring Cloud Config Server supports decrypting properties from Vault by utilizing a special placeholder prefix `\{vault}`. This feature allows for dynamic resolution of sensitive configuration properties directly from Vault at runtime.
+
+=== Configuration Steps
+
+All configuration settings for integrating with Vault should be placed in your `application.yml`. Below are the specific configurations required to activate the Vault profile, connect to your Vault server, and format properties using the `\{vault}` prefix.
```

Comment:
It could be in application.yaml or application.properties so maybe we say `application.[yaml | properties]?